.github/workflows/overcloud-host-image-build.yml

```
@@ -271,7 +284,8 @@ jobs:
           kayobe playbook run \
           src/kayobe-config/etc/kayobe/ansible/openstack-host-image-upload.yml \
           -e local_image_path="/opt/kayobe/images/overcloud-rocky-9/overcloud-rocky-9.qcow2" \
-          -e image_name=overcloud-rocky-9-${{ steps.host_image_tag.outputs.host_image_tag }}
+          -e image_name=overcloud-rocky-9-${{ steps.host_image_tag.outputs.host_image_tag }} \
+          -e cpu-platform=${{ env.overcloud_dib_architecture }}
```

Ansible variable names with dashes can cause all sorts of problems
```suggestion
          -e cpu_platform=${{ env.overcloud_dib_architecture }}
```